File: VB/WindowsApplication1/Custom style/MyRegistrator.vb
Imports System.ComponentModel
Imports System.Drawing
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.Drawing

Namespace WindowsApplication1

    Public Class MyRegistrator
        Inherits SkinViewInfoRegistrator

        Public Sub New()
        End Sub

        Public Overrides ReadOnly Property ViewName As String
            Get
                Return "MyStyle"
            End Get
        End Property

        Public Overrides Function CreatePainter(ByVal tabControl As DevExpress.XtraTab.IXtraTab) As DevExpress.XtraTab.Drawing.BaseTabPainter
            Return New MySkinTabPainter(tabControl)
        End Function
    End Class
End Namespace

File: CS/WindowsApplication1/Custom style/MyRegistrator.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.Drawing;

namespace WindowsApplication1
{
    public class MyRegistrator : SkinViewInfoRegistrator
    {

        public MyRegistrator()
        {

        }

        public override string ViewName
        {
            get
            {
                return "MyStyle";
            }
        }

        public override DevExpress.XtraTab.Drawing.BaseTabPainter CreatePainter(DevExpress.XtraTab.IXtraTab tabControl)
        {
            return new MySkinTabPainter(tabControl);
        }
    }
}
